### PR TITLE
readme: include links to our social (twitter + discord)

### PR DESCRIPTION
**Pull request by gneyal**

### CODE REVIEW

**Comment by gneyal** [comment]
Changed the readme page. Links to bugs and new features will be addressed after new templates will be in place.
Screen Shot 2022-08-31 at 14 00 09

File: README.md [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
+          <a href="https://github.com/ariga/atlas/issues">Report a Bug</a>
+    ·
+          <a href="https://github.com/ariga/atlas/discussions">Request a Feature</a>
+     </p>
+          
+
+</div>
+
+
+# What is it?
```
Comment (a8m) [inline README.md:30]: Feels redundant to me.

**Review by a8m — APPROVED** [review]
Looks good, but I added a few suggestions.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,33 @@
...
+<div align="center">
+Manage your database schemas with Atlas CLI
+</div>
+
+<div align="center">
+     <p align="center">
+         <br />
+         <a href="https://atlasgo.io/getting-started/" rel="dofollow"><strong>Explore the docs »</strong></a>
+         <br /><br/>
+          <a href="https://discord.com/invite/zZ6sWVg6NT">Discord</a>
```
[the remark this answers is not in the document]
Comment (gneyal) [reply README.md:17]: Explore the docs is the main call to action. 

I think that the links in the second line are secondary to what we want from people. It's not about saving space, it's about highlighting the important stuff.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,31 @@
...
+          <a href="https://twitter.com/ariga_io/">Twitter</a>
+     ·
+          <a href="https://github.com/ariga/atlas/issues">Report a Bug</a>
+    ·
+          <a href="https://github.com/ariga/atlas/discussions">Request a Feature</a>
+     </p>
+</div>
+
+
+# What is it?
```
Comment (a8m) [inline README.md:28]:
```suggestion
```

**Pull request merged**